rollup/node/src/chain_spec.rs

```diff
@@ -321,5 +323,11 @@ fn rollup_genesis(
 			},
 		},
 		rolldown: rollup_runtime::RolldownConfig { _phantom: Default::default() },
+		metamask: rollup_runtime::MetamaskConfig {
+			name: "Gasp".to_string(),
+			version: "0.0.1".to_string(),
+			chain_id: 17000u64,
```

Comment:
Don't you want to use eth_chain_id here?

Reply:
:man_facepalming: